poetry.lock

```diff
@@ -500,14 +487,13 @@ diagrams = ["jinja2", "railroad-diagrams"]
 
 [[package]]
 name = "pysigma"
-version = "0.9.9"
+version = "0.9.11"
```

Comment:
The new lock file is still updating pySigma to 0.9.11. If we lock pySigma to 0.9.9 for the time being in the pyproject.toml, it should avoid this issue.